tests/CodeSniffer/Helper/FileHelper.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BestIt\CodeSniffer\Helper;
+
+use PHP_CodeSniffer\Config;
+use PHP_CodeSniffer\Exceptions\RuntimeException;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Ruleset;
+use function file_get_contents;
+
+/**
+ * Helps the tests to load a code sniffer file.
+ *
+ * @author blange <bjoern.lange@bestit-online.de>
+ * @package BestIt\CodeSniffer\Helper
+ */
+class FileHelper
+{
+    /**
+     * Fills a codesniffer file object for the given file path.
+     *
+     * @param string $filePath
```

Comment:
Well, shouldn't the codesniffer pass all own rules? In this case the argument description is missing ...

I know it's just a warning and a lot of projects ignore this but the codesniffer self should pass all rules - or no?